### PR TITLE
Add support for XTDB multinode

### PR DESCRIPTION
**Pull request by dekkers**
Add automatic creation and deletion of nodes octopoes/xtdb and organization in katalogus.

A few things are moved around to prevent circular imports between katalogus/client.py and tools/models.py.

The error messages in the API are formatted using drf-standardized-errors.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -31,6 +31,7 @@ django-password-validators==1.7.0
 django-csp==3.7
 djangorestframework==3.14.0
 django-tagulous==1.3.3
+drf-standardized-errors==0.12.4
```
Comment (ammar92) [inline requirements-dev.txt:34]: If this package is already included in `requirements.txt` then it's not needed here anymore, right?

File: tests/conftest.py [hunk 2]
```diff
@@ -14,3 +19,13 @@ def user(django_user_model):
     user = django_user_model.objects.create_user(email="admin@openkat.nl", password="TestTest123!!")
     user.is_verified = lambda: True
     return user
+
+
+@pytest.fixture
+def mock_katalogus(mocker):
```
Comment (ammar92) [inline tests/conftest.py:25]: Handy 👍

File: tests/test_api_organization.py [hunk 3]
```diff
@@ -88,6 +103,58 @@ def test_it_returns_organization(self, json):
...
+                "type": "server_error",
+                "errors": [
+                    {
+                        "code": "error",
+                        "detail": "Octopoes returned error creating organization: Test error",
+                        "attr": None,
+                    }
+                ],
+            }
+            assert expected == json
```
Comment (ammar92) [inline tests/test_api_organization.py:156]: The order in pytest is usually `assert actual == expected`, this way it assigns the correct value to 'actual' and 'expected' in the logs

```suggestion
            assert json == expected
```

**Review by ammar92 — APPROVED** [review]
Cool stuff. I really like the tests and exception handling. Few remarks but consider it approved

File: requirements-dev.txt (hunk 1 above)
Comment (Darwinkel) [reply requirements-dev.txt:34]: Some commands/environments only use `requirements-dev.txt`, so at the moment it is required. I guess we can refactor that in a future PR.

Comment (Darwinkel) [reply requirements-dev.txt:34]: Some commands/environments only use `requirements-dev.txt`, so at the moment it is required. I guess we can refactor that in a future PR.

Comment (Darwinkel) [reply requirements-dev.txt:34]: Some commands/environments only use `requirements-dev.txt`, so at the moment it is required. I guess we can refactor that in a future PR.

**Comment by Darwinkel** [comment]
No regressions in conjunction with https://github.com/minvws/nl-kat-coordination/pull/92.

**Review by Darwinkel — APPROVED** [review]

**Review by underdarknl — APPROVED** [review]

**Pull request merged**